Acceptance tests fail verifying NFT transactions REST API

### Description

Acceptance tests regularly fail when running the `Validate Base NFT Flow - Create, Associate, Mint, Transfer` test. The NFT history API will return 206 for partial content before the transaction shows up, which our retry logic does not account for. You can see this if you try a valid token but with a non-existing serial number:

curl -vf "http://testnet.mirrornode.hedera.com/api/v1/tokens/0.0.2317992/nfts/9999999/transactions"

### Steps to reproduce

`./mvnw --batch-mode clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags=@critical -Dspring.config.name=application-templated`

### Additional context

```
  Then I mint a serial number from the token                                                         # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.mintNftToken()
2021-08-30T13:54:10.600-0600 DEBUG main o.s.r.s.RetryTemplate Retry: count=0 
2021-08-30T13:54:10.706-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientConnect [id:e806ca7c-1, L:/10.44.12.27:41654 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:10.824-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientOperations [id:e806ca7c-1, L:/10.44.12.27:41654 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"32-tmZWC8aWVU5PFmc1wf+KD2xm+SI", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:10 GMT, Connection=close, content-length=50] 
2021-08-30T13:54:10.826-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientOperations [id:e806ca7c-1, L:/10.44.12.27:41654 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:10.827-0600 WARN reactor-tcp-epoll-4 c.h.m.t.e.a.c.MirrorNodeClient Retry attempt #1 after failure: 404 Not Found from GET http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597 
2021-08-30T13:54:11.216-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientConnect [id:73c4a6fa-1, L:/10.44.12.27:41656 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:11.334-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientOperations [id:73c4a6fa-1, L:/10.44.12.27:41656 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"32-tmZWC8aWVU5PFmc1wf+KD2xm+SI", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:11 GMT, Connection=close, content-length=50] 
2021-08-30T13:54:11.335-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientOperations [id:73c4a6fa-1, L:/10.44.12.27:41656 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:11.336-0600 WARN reactor-tcp-epoll-1 c.h.m.t.e.a.c.MirrorNodeClient Retry attempt #2 after failure: 404 Not Found from GET http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597 
2021-08-30T13:54:12.099-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientConnect [id:b159f33d-1, L:/10.44.12.27:41660 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:12.216-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientOperations [id:b159f33d-1, L:/10.44.12.27:41660 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"32-tmZWC8aWVU5PFmc1wf+KD2xm+SI", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:12 GMT, Connection=close, content-length=50] 
2021-08-30T13:54:12.217-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientOperations [id:b159f33d-1, L:/10.44.12.27:41660 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:12.218-0600 WARN reactor-tcp-epoll-2 c.h.m.t.e.a.c.MirrorNodeClient Retry attempt #3 after failure: 404 Not Found from GET http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597 
2021-08-30T13:54:13.153-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientConnect [id:90046b86-1, L:/10.44.12.27:41664 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:13.272-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientOperations [id:90046b86-1, L:/10.44.12.27:41664 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"32-tmZWC8aWVU5PFmc1wf+KD2xm+SI", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:13 GMT, Connection=close, content-length=50] 
2021-08-30T13:54:13.273-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientOperations [id:90046b86-1, L:/10.44.12.27:41664 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:13.274-0600 WARN reactor-tcp-epoll-3 c.h.m.t.e.a.c.MirrorNodeClient Retry attempt #4 after failure: 404 Not Found from GET http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597 
2021-08-30T13:54:16.062-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientConnect [id:c4220dd8-1, L:/10.44.12.27:41674 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:16.183-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientOperations [id:c4220dd8-1, L:/10.44.12.27:41674 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"2e9-lPJsiYjxy99Me1KKgap+HSYUo8g", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:16 GMT, Connection=close, content-length=745] 
2021-08-30T13:54:16.184-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientOperations [id:c4220dd8-1, L:/10.44.12.27:41674 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:16.291-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientConnect [id:77bac445-1, L:/10.44.12.27:41676 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/tokens/0.0.2317992, method=GET} 
2021-08-30T13:54:16.410-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientOperations [id:77bac445-1, L:/10.44.12.27:41676 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"3c5-jE4H4NYlPK12qSUOpay6F6Yf8mk", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:16 GMT, Connection=close, content-length=965] 
2021-08-30T13:54:16.411-0600 DEBUG reactor-tcp-epoll-1 r.n.h.c.HttpClientOperations [id:77bac445-1, L:/10.44.12.27:41676 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:16.517-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientConnect [id:4ff7411f-1, L:/10.44.12.27:41680 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/tokens/0.0.2317992/nfts/1, method=GET} 
2021-08-30T13:54:16.636-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientOperations [id:4ff7411f-1, L:/10.44.12.27:41680 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"bf-VyGoCp1biXyTa29hXeStl2jt3SY", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:16 GMT, Connection=close, content-length=191] 
2021-08-30T13:54:16.638-0600 DEBUG reactor-tcp-epoll-2 r.n.h.c.HttpClientOperations [id:4ff7411f-1, L:/10.44.12.27:41680 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:16.744-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientConnect [id:389c9d44-1, L:/10.44.12.27:41682 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/transactions/0.0.88-1630353237-202574597, method=GET} 
2021-08-30T13:54:16.863-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientOperations [id:389c9d44-1, L:/10.44.12.27:41682 - R:35.204.41.101/35.204.41.101:80] Received response (auto-read:false) : [Access-Control-Allow-Origin=*, Content-Type=application/json; charset=utf-8, ETag=W/"2e9-lPJsiYjxy99Me1KKgap+HSYUo8g", Vary=Accept-Encoding, Date=Mon, 30 Aug 2021 19:54:16 GMT, Connection=close, content-length=745] 
2021-08-30T13:54:16.864-0600 DEBUG reactor-tcp-epoll-3 r.n.h.c.HttpClientOperations [id:389c9d44-1, L:/10.44.12.27:41682 - R:35.204.41.101/35.204.41.101:80] Received last HTTP packet 
2021-08-30T13:54:16.974-0600 DEBUG reactor-tcp-epoll-4 r.n.h.c.HttpClientConnect [id:c98b4ad7-1, L:/10.44.12.27:41684 - R:35.204.41.101/35.204.41.101:80] Handler is being applied: {uri=http://35.204.41.101/api/v1/tokens/0.0.2317992/nfts/1/transactions, method=GET} 
2021-08-30T13:54:21.977-0600 WARN reactor-tcp-epoll-4 r.n.h.c.HttpClientConnect [id:c98b4ad7-1, L:/10.44.12.27:41684 - R:35.204.41.101/35.204.41.101:80] The connection observed an error io.netty.handler.timeout.ReadTimeoutException
2021-08-30T13:54:21.980-0600 ERROR reactor-tcp-epoll-4 c.h.m.t.e.a.c.MirrorNodeClient Endpoint failed, returning: nested exception is io.netty.handler.timeout.ReadTimeoutException 
2021-08-30T13:54:21.981-0600 DEBUG main o.s.r.s.RetryTemplate Checking for rethrow: count=1 
2021-08-30T13:54:21.981-0600 DEBUG main o.s.r.s.RetryTemplate Retry failed last attempt: count=1 
  And the mirror node REST API should return status 200 for token 0 serial number 0 full flow        # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyMirrorNftFundFlow(int,java.lang.Integer,java.lang.Integer)
      org.springframework.web.reactive.function.client.WebClientRequestException: nested exception is io.netty.handler.timeout.ReadTimeoutException
	at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
	at ???.the mirror node REST API should return status 200 for token 0 serial number 0 full flow(file:///home/jenkins/agent/workspace/hcs-acceptance/hedera-mirror-test/src/test/resources/features/hts/hts.feature:67)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	|_ checkpoint ??? Request to GET http://35.204.41.101/api/v1/tokens/0.0.2317992/nfts/1/transactions [DefaultWebClient]
Stack trace:
		at org.springframework.web.reactive.function.client.ExchangeFunctions$DefaultExchangeFunction.lambda$wrapException$9(ExchangeFunctions.java:141)
		at reactor.core.publisher.MonoErrorSupplied.subscribe(MonoErrorSupplied.java:55)
		at reactor.core.publisher.Mono.subscribe(Mono.java:4338)
		at reactor.core.publisher.FluxOnErrorResume$ResumeSubscriber.onError(FluxOnErrorResume.java:103)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.FluxPeek$PeekSubscriber.onError(FluxPeek.java:222)
		at reactor.core.publisher.MonoNext$NextSubscriber.onError(MonoNext.java:93)
		at reactor.core.publisher.MonoFlatMapMany$FlatMapManyMain.onError(MonoFlatMapMany.java:204)
		at reactor.core.publisher.SerializedSubscriber.onError(SerializedSubscriber.java:124)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.whenError(FluxRetryWhen.java:225)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenOtherSubscriber.onError(FluxRetryWhen.java:274)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.drain(FluxConcatMap.java:414)
		at reactor.core.publisher.FluxConcatMap$ConcatMapImmediate.onNext(FluxConcatMap.java:251)
		at reactor.core.publisher.EmitterProcessor.drain(EmitterProcessor.java:491)
		at reactor.core.publisher.EmitterProcessor.tryEmitNext(EmitterProcessor.java:299)
		at reactor.core.publisher.SinkManySerialized.tryEmitNext(SinkManySerialized.java:100)
		at reactor.core.publisher.InternalManySink.emitNext(InternalManySink.java:27)
		at reactor.core.publisher.FluxRetryWhen$RetryWhenMainSubscriber.onError(FluxRetryWhen.java:190)
		at reactor.core.publisher.MonoCreate$DefaultMonoSink.error(MonoCreate.java:189)
		at reactor.netty.http.client.HttpClientConnect$HttpObserver.onUncaughtException(HttpClientConnect.java:384)
		at reactor.netty.ReactorNetty$CompositeConnectionObserver.onUncaughtException(ReactorNetty.java:647)
		at reactor.netty.resources.DefaultPooledConnectionProvider$DisposableAcquire.onUncaughtException(DefaultPooledConnectionProvider.java:219)
		at reactor.netty.resources.DefaultPooledConnectionProvider$PooledConnection.onUncaughtException(DefaultPooledConnectionProvider.java:467)
		at reactor.netty.channel.FluxReceive.drainReceiver(FluxReceive.java:232)
		at reactor.netty.channel.FluxReceive.onInboundError(FluxReceive.java:453)
		at reactor.netty.channel.ChannelOperations.onInboundError(ChannelOperations.java:488)
		at reactor.netty.channel.ChannelOperationsHandler.exceptionCaught(ChannelOperationsHandler.java:126)
		at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:302)
		at io.netty.channel.AbstractChannelHandlerContext.invokeExceptionCaught(AbstractChannelHandlerContext.java:281)
		at io.netty.channel.AbstractChannelHandlerContext.fireExceptionCaught(AbstractChannelHandlerContext.java:273)
		at io.netty.handler.timeout.ReadTimeoutHandler.readTimedOut(ReadTimeoutHandler.java:98)
		at io.netty.handler.timeout.ReadTimeoutHandler.channelIdle(ReadTimeoutHandler.java:90)
		at io.netty.handler.timeout.IdleStateHandler$ReaderIdleTimeoutTask.run(IdleStateHandler.java:506)
		at io.netty.handler.timeout.IdleStateHandler$AbstractIdleTask.run(IdleStateHandler.java:478)
		at io.netty.util.concurrent.PromiseTask.runTask(PromiseTask.java:98)
		at io.netty.util.concurrent.ScheduledFutureTask.run(ScheduledFutureTask.java:170)
		at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
		at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
		at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:384)
		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.base/java.lang.Thread.run(Thread.java:835)
	Suppressed: java.lang.Exception: #block terminated with an error
		at reactor.core.publisher.BlockingSingleSubscriber.blockingGet(BlockingSingleSubscriber.java:99)
		at reactor.core.publisher.Mono.block(Mono.java:1704)
		at com.hedera.mirror.test.e2e.acceptance.client.MirrorNodeClient.callRestEndpoint(MirrorNodeClient.java:168)
		at com.hedera.mirror.test.e2e.acceptance.client.MirrorNodeClient.getNftTransactions(MirrorNodeClient.java:139)
		at com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyNftTransactions(TokenFeature.java:585)
		at com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyMirrorNftFundFlow(TokenFeature.java:362)
		at com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature$$FastClassBySpringCGLIB$$121a4324.invoke(<generated>)
		at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
		at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:779)
		at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
		at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750)
		at org.springframework.retry.interceptor.RetryOperationsInterceptor$1.doWithRetry(RetryOperationsInterceptor.java:93)
		at org.springframework.retry.support.RetryTemplate.doExecute(RetryTemplate.java:329)
		at org.springframework.retry.support.RetryTemplate.execute(RetryTemplate.java:209)
		at org.springframework.retry.interceptor.RetryOperationsInterceptor.invoke(RetryOperationsInterceptor.java:119)
		at org.springframework.retry.annotation.AnnotationAwareRetryOperationsInterceptor.invoke(AnnotationAwareRetryOperationsInterceptor.java:163)
		at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
		at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:750)
		at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:692)
		at com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature$$EnhancerBySpringCGLIB$$e171ced3.verifyMirrorNftFundFlow(<generated>)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:567)
		at io.cucumber.java.Invoker.doInvoke(Invoker.java:66)
		at io.cucumber.java.Invoker.invoke(Invoker.java:24)
		at io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)
		at io.cucumber.java.JavaStepDefinition.execute(JavaStepDefinition.java:29)
		at io.cucumber.core.runner.CoreStepDefinition.execute(CoreStepDefinition.java:66)
		at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:63)
		at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
		at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:92)
		at io.cucumber.core.runner.TestStep.run(TestStep.java:64)
		at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
		at io.cucumber.core.runner.TestCase.run(TestCase.java:104)
		at io.cucumber.core.runner.Runner.runPickle(Runner.java:73)
		at io.cucumber.junit.PickleRunners$NoStepDescriptions.run(PickleRunners.java:151)
		at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:135)
		at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:27)
		at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
		at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
		at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
		at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
		at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
		at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
		at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
		at io.cucumber.junit.Cucumber.runChild(Cucumber.java:200)
		at io.cucumber.junit.Cucumber.runChild(Cucumber.java:90)
		at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
		at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
		at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
		at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
		at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
		at io.cucumber.junit.Cucumber$RunCucumber.evaluate(Cucumber.java:235)
		at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
		at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
		at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
		at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
		at org.junit.vintage.engine.execution.RunnerExecutor.execute(RunnerExecutor.java:43)
		at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
		at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
		at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
		at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
		at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
		at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
		at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
		at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
		at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
		at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
		at org.junit.vintage.engine.VintageTestEngine.executeAllChildren(VintageTestEngine.java:82)
		at org.junit.vintage.engine.VintageTestEngine.execute(VintageTestEngine.java:73)
		at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
		at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
		at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
		at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
		at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
		at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
		at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:75)
		at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.execute(JUnitPlatformProvider.java:188)
		at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:154)
		at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:128)
		at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
		at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
		at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
		at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: io.netty.handler.timeout.ReadTimeoutException

  Then I transfer serial number of token to recipient                                                # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.transferNftsToRecipient(java.lang.Integer,java.lang.Integer,java.lang.Integer)
  And the mirror node REST API should return status 200 for token 0 serial number 0 transaction flow # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyMirrorNftTransactionsAPIResponses(int,java.lang.Integer,java.lang.Integer)
```

### Hedera network

testnet

### Version

v0.39.0

### Operating system

_No response_

--- a/hedera-mirror-test/README.md
+++ b/hedera-mirror-test/README.md
@@ -77,6 +77,11 @@ under `hedera.mirror.test.acceptance` include:
   nodes. Populating `hedera.mirror.test.acceptance.nodes` will take priority over this.
 - `sdkProperties`
   - `maxAttempts` - The maximum number of times the sdk should try to submit a transaction to the network.
+- `webclient`
+  - `connectionTimeout` - The timeout duration to wait to establish a connection with the server
+  - `readTimeout` - The timeout duration to wait for data to be read.
+  - `wiretap` - Whether a wire logger configuration should be applied to connection calls.
+  - `writeTimeout` - The timeout duration to wait for data to be written.
 
 (Recommended) Options can be set by creating your own configuration file with the above properties. This allows for
 multiple files per env. The following command will help to point out which file to use:

--- a/hedera-mirror-test/README.md
+++ b/hedera-mirror-test/README.md
@@ -123,11 +123,11 @@ tag are run. To run a different set of files different tags can be specified
 
 Test Suite Tags
 
-- `@critical` - Test cases to ensure the network is up and running and satisfies base scenarios. Total cost to run 31.6
+- `@critical` - Test cases to ensure the network is up and running and satisfies base scenarios. Total cost to run 6.5
   ℏ.
 - `@release` - Test cases to verify a new deployed version satisfies core scenarios and is release worthy. Total cost to
   run 19.2 ℏ.
-- `@acceptance` - Test cases to verify most feature scenarios meet customer acceptance criteria. Total cost to run 6.5
+- `@acceptance` - Test cases to verify most feature scenarios meet customer acceptance criteria. Total cost to run 31.6
   ℏ.
 - `@fullsuite` - All cases - this will require some configuration of feature files and may include some disabled tests
   that will fail on execution. Total cost to run 33.9 ℏ.

--- a/hedera-mirror-test/README.md
+++ b/hedera-mirror-test/README.md
@@ -73,6 +73,7 @@ under `hedera.mirror.test.acceptance` include:
   - `maxAttempts` - The maximum number of attempts when calling a REST API endpoint and receiving a 404.
   - `maxBackoff` - The maximum backoff duration the mirror grpc subscriber will wait between attempts.
   - `minBackoff` - The minimum backoff duration the mirror grpc subscriber will wait between attempts.
+  - `retryableExceptions` - List of retryable exception class types
 - `retrieveAddressBook` - Whether to download the address book from the network and use those nodes over the default
   nodes. Populating `hedera.mirror.test.acceptance.nodes` will take priority over this.
 - `sdkProperties`

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/AccountClient.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/AccountClient.java
@@ -114,7 +114,7 @@ public class AccountClient extends AbstractNetworkClient {
                 .setTransactionMemo("transfer test");
     }
 
-    public TransactionReceipt sendCryptoTransfer(AccountId recipient, Hbar hbarAmount) {
+    public NetworkTransactionResponse sendCryptoTransfer(AccountId recipient, Hbar hbarAmount) {
         log.debug(
                 "Send CryptoTransfer of {} tℏ from {} to {}", hbarAmount.toTinybars(),
                 sdkClient.getExpandedOperatorAccountId().getAccountId(),
@@ -123,12 +123,12 @@ public class AccountClient extends AbstractNetworkClient {
         TransferTransaction cryptoTransferTransaction = getCryptoTransferTransaction(sdkClient
                 .getExpandedOperatorAccountId().getAccountId(), recipient, hbarAmount);
 
-        TransactionReceipt transactionReceipt = executeTransactionAndRetrieveReceipt(cryptoTransferTransaction)
-                .getReceipt();
+        NetworkTransactionResponse networkTransactionResponse =
+                executeTransactionAndRetrieveReceipt(cryptoTransferTransaction);
 
         log.debug("Sent CryptoTransfer");
 
-        return transactionReceipt;
+        return networkTransactionResponse;
     }
 
     public AccountCreateTransaction getAccountCreateTransaction(Hbar initialBalance, KeyList publicKeys,

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/AccountClient.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/AccountClient.java
@@ -101,7 +101,7 @@ public class AccountClient extends AbstractNetworkClient {
                 .execute(client)
                 .hbars;
 
-        log.debug("{} balance is {}", accountId, balance);
+        log.info("{} balance is {}", accountId, balance);
 
         return balance.toTinybars();
     }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
@@ -27,7 +27,6 @@ import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 import javax.inject.Named;
 import lombok.extern.log4j.Log4j2;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.retry.support.RetryTemplate;
 import org.springframework.web.reactive.function.client.WebClient;
@@ -180,7 +179,6 @@ public class MirrorNodeClient extends AbstractNetworkClient {
                 t instanceof ReadTimeoutException ||
                 t.getCause() instanceof ReadTimeoutException ||
                 t instanceof TimeoutException ||
-                (t instanceof WebClientResponseException &&
-                        ((WebClientResponseException) t).getStatusCode() == HttpStatus.NOT_FOUND);
+                t instanceof WebClientResponseException;
     }
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
@@ -21,7 +21,7 @@ package com.hedera.mirror.test.e2e.acceptance.client;
  */
 
 import com.google.common.base.Stopwatch;
-import io.grpc.netty.shaded.io.netty.handler.timeout.ReadTimeoutException;
+import io.netty.handler.timeout.ReadTimeoutException;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
@@ -178,6 +178,7 @@ public class MirrorNodeClient extends AbstractNetworkClient {
     protected boolean shouldRetryRestCall(Throwable t) {
         return t instanceof PrematureCloseException ||
                 t instanceof ReadTimeoutException ||
+                t.getCause() instanceof ReadTimeoutException ||
                 t instanceof TimeoutException ||
                 (t instanceof WebClientResponseException &&
                         ((WebClientResponseException) t).getStatusCode() == HttpStatus.NOT_FOUND);

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/MirrorNodeClient.java
@@ -21,17 +21,14 @@ package com.hedera.mirror.test.e2e.acceptance.client;
  */
 
 import com.google.common.base.Stopwatch;
-import io.netty.handler.timeout.ReadTimeoutException;
+import com.google.common.base.Throwables;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
 import javax.inject.Named;
 import lombok.extern.log4j.Log4j2;
 import org.springframework.http.MediaType;
 import org.springframework.retry.support.RetryTemplate;
 import org.springframework.web.reactive.function.client.WebClient;
-import org.springframework.web.reactive.function.client.WebClientResponseException;
-import reactor.netty.http.client.PrematureCloseException;
 import reactor.util.retry.Retry;
 import reactor.util.retry.RetryBackoffSpec;
 
@@ -175,10 +172,8 @@ public class MirrorNodeClient extends AbstractNetworkClient {
     }
 
     protected boolean shouldRetryRestCall(Throwable t) {
-        return t instanceof PrematureCloseException ||
-                t instanceof ReadTimeoutException ||
-                t.getCause() instanceof ReadTimeoutException ||
-                t instanceof TimeoutException ||
-                t instanceof WebClientResponseException;
+        return sdkClient.getAcceptanceTestProperties().getRestPollingProperties().getRetryableExceptions()
+                .stream()
+                .anyMatch(ex -> ex.isInstance(t) || ex.isInstance(Throwables.getRootCause(t)));
     }
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/AcceptanceTestProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/AcceptanceTestProperties.java
@@ -75,6 +75,8 @@ public class AcceptanceTestProperties {
 
     private final SdkProperties sdkProperties;
 
+    private final WebClientProperties webClientProperties;
+
     public Set<NodeProperties> getNodes() {
         if (network == HederaNetwork.OTHER && nodes.isEmpty()) {
             throw new IllegalArgumentException("nodes must not be empty");

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/ClientConfiguration.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/ClientConfiguration.java
@@ -22,7 +22,7 @@ package com.hedera.mirror.test.e2e.acceptance.config;
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import io.netty.channel.ChannelOption;
 import io.netty.handler.timeout.ReadTimeoutHandler;
 import io.netty.handler.timeout.WriteTimeoutHandler;
@@ -44,7 +44,6 @@ import org.springframework.retry.policy.SimpleRetryPolicy;
 import org.springframework.retry.support.RetryTemplate;
 import org.springframework.web.reactive.function.client.WebClient;
 import reactor.netty.http.client.HttpClient;
-import reactor.netty.tcp.TcpClient;
 
 import com.hedera.hashgraph.sdk.PrecheckStatusException;
 import com.hedera.hashgraph.sdk.ReceiptStatusException;
@@ -79,24 +78,30 @@ class ClientConfiguration {
 
     @Bean
     WebClient webClient() {
-        TcpClient tcpClient = TcpClient.create()
-                .option(ChannelOption.CONNECT_TIMEOUT_MILLIS, 5000)
+        HttpClient httpClient = HttpClient.create()
+                .option(
+                        ChannelOption.CONNECT_TIMEOUT_MILLIS,
+                        acceptanceTestProperties.getWebClientProperties().getConnectTimeoutMillis())
                 .doOnConnected(connection -> {
-                    connection.addHandlerLast(new ReadTimeoutHandler(5000, TimeUnit.MILLISECONDS));
-                    connection.addHandlerLast(new WriteTimeoutHandler(5000, TimeUnit.MILLISECONDS));
+                    connection.addHandlerLast(new ReadTimeoutHandler(
+                            acceptanceTestProperties.getWebClientProperties().getConnectionReadTimeoutMillis(),
+                            TimeUnit.MILLISECONDS));
+                    connection.addHandlerLast(new WriteTimeoutHandler(
+                            acceptanceTestProperties.getWebClientProperties().getConnectionWriteTimeoutMillis(),
+                            TimeUnit.MILLISECONDS));
                 })
-                .wiretap(true); // enable request logging
+                .wiretap(acceptanceTestProperties.getWebClientProperties().isWiretap()); // enable request logging
 
         // support snake_case to avoid manually mapping JsonProperty on all properties
         ObjectMapper objectMapper = new ObjectMapper()
                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-                .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
+                .setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE);
         Jackson2JsonDecoder jackson2JsonDecoder = new Jackson2JsonDecoder(objectMapper, MediaType.APPLICATION_JSON);
         Jackson2JsonEncoder jackson2JsonEncoder = new Jackson2JsonEncoder(objectMapper, MediaType.APPLICATION_JSON);
 
         return WebClient.builder()
                 .baseUrl(acceptanceTestProperties.getRestPollingProperties().getBaseUrl())
-                .clientConnector(new ReactorClientHttpConnector(HttpClient.from(tcpClient)))
+                .clientConnector(new ReactorClientHttpConnector(httpClient))
                 .codecs(clientCodecConfigurer -> {
                     clientCodecConfigurer.defaultCodecs().jackson2JsonDecoder(jackson2JsonDecoder);
                     clientCodecConfigurer.defaultCodecs().jackson2JsonEncoder(jackson2JsonEncoder);

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/ClientConfiguration.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/ClientConfiguration.java
@@ -81,13 +81,13 @@ class ClientConfiguration {
         HttpClient httpClient = HttpClient.create()
                 .option(
                         ChannelOption.CONNECT_TIMEOUT_MILLIS,
-                        acceptanceTestProperties.getWebClientProperties().getConnectTimeoutMillis())
+                        (int) acceptanceTestProperties.getWebClientProperties().getConnectionTimeout().toMillis())
                 .doOnConnected(connection -> {
                     connection.addHandlerLast(new ReadTimeoutHandler(
-                            acceptanceTestProperties.getWebClientProperties().getConnectionReadTimeoutMillis(),
+                            acceptanceTestProperties.getWebClientProperties().getReadTimeout().toMillis(),
                             TimeUnit.MILLISECONDS));
                     connection.addHandlerLast(new WriteTimeoutHandler(
-                            acceptanceTestProperties.getWebClientProperties().getConnectionWriteTimeoutMillis(),
+                            acceptanceTestProperties.getWebClientProperties().getWriteTimeout().toMillis(),
                             TimeUnit.MILLISECONDS));
                 })
                 .wiretap(acceptanceTestProperties.getWebClientProperties().isWiretap()); // enable request logging

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
@@ -21,6 +21,7 @@ package com.hedera.mirror.test.e2e.acceptance.config;
  */
 
 import java.time.Duration;
+import java.util.Set;
 import javax.validation.constraints.Max;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotBlank;
@@ -51,4 +52,6 @@ public class RestPollingProperties {
     @NotNull
     @DurationMin(millis = 100L)
     private Duration minBackoff = Duration.ofMillis(250L);
+
+    private Set<Class> retryableExceptions = Set.of(Exception.class);
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/RestPollingProperties.java
@@ -53,5 +53,6 @@ public class RestPollingProperties {
     @DurationMin(millis = 100L)
     private Duration minBackoff = Duration.ofMillis(250L);
 
+    @NotNull
     private Set<Class> retryableExceptions = Set.of(Exception.class);
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
@@ -1,0 +1,48 @@
+package com.hedera.mirror.test.e2e.acceptance.config;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+
+@Component
+@ConfigurationProperties(prefix = "hedera.mirror.test.acceptance.webclient")
+@Data
+@Validated
+public class WebClientProperties {
+    @Min(5000)
+    @Max(60000)
+    private int connectionReadTimeoutMillis = 10000;
+
+    @Min(5000)
+    @Max(60000)
+    private int connectionWriteTimeoutMillis = 10000;
+
+    @Min(5000)
+    @Max(60000)
+    private int connectTimeoutMillis = 10000;
+
+    private boolean wiretap = false;
+}

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
@@ -20,9 +20,11 @@ package com.hedera.mirror.test.e2e.acceptance.config;
  * ‍
  */
 
-import javax.validation.constraints.Max;
-import javax.validation.constraints.Min;
+import java.time.Duration;
+import javax.validation.constraints.NotNull;
 import lombok.Data;
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.stereotype.Component;
 import org.springframework.validation.annotation.Validated;
@@ -32,17 +34,20 @@ import org.springframework.validation.annotation.Validated;
 @Data
 @Validated
 public class WebClientProperties {
-    @Min(5000)
-    @Max(60000)
-    private int connectionReadTimeoutMillis = 10000;
+    @NotNull
+    @DurationMin(millis = 5000L)
+    @DurationMax(millis = 60000L)
+    private Duration connectionTimeout = Duration.ofSeconds(10L);
 
-    @Min(5000)
-    @Max(60000)
-    private int connectionWriteTimeoutMillis = 10000;
-
-    @Min(5000)
-    @Max(60000)
-    private int connectTimeoutMillis = 10000;
+    @NotNull
+    @DurationMin(millis = 5000L)
+    @DurationMax(millis = 10000L)
+    private Duration readTimeout = Duration.ofSeconds(10L);
 
     private boolean wiretap = false;
+
+    @NotNull
+    @DurationMin(millis = 5000L)
+    @DurationMax(millis = 10000L)
+    private Duration writeTimeout = Duration.ofSeconds(10L);
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/config/WebClientProperties.java
@@ -35,19 +35,19 @@ import org.springframework.validation.annotation.Validated;
 @Validated
 public class WebClientProperties {
     @NotNull
-    @DurationMin(millis = 5000L)
-    @DurationMax(millis = 60000L)
+    @DurationMin(seconds = 5L)
+    @DurationMax(seconds = 60L)
     private Duration connectionTimeout = Duration.ofSeconds(10L);
 
     @NotNull
-    @DurationMin(millis = 5000L)
-    @DurationMax(millis = 10000L)
+    @DurationMin(seconds = 5L)
+    @DurationMax(seconds = 60L)
     private Duration readTimeout = Duration.ofSeconds(10L);
 
     private boolean wiretap = false;
 
     @NotNull
-    @DurationMin(millis = 5000L)
-    @DurationMax(millis = 10000L)
+    @DurationMin(seconds = 5L)
+    @DurationMax(seconds = 60L)
     private Duration writeTimeout = Duration.ofSeconds(10L);
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
@@ -60,9 +60,6 @@ public class AccountFeature {
     @When("I request balance info for this account")
     public void getAccountBalance() {
         balance = accountClient.getBalance();
-        log.info("{} account balance is {}",
-                accountClient.getSdkClient().getExpandedOperatorAccountId().getAccountId(),
-                balance);
     }
 
     @Then("the crypto balance should be greater than or equal to {long}")

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
@@ -121,21 +121,17 @@ public class AccountFeature {
     }
 
     @Then("the mirror node REST API should return status {int} for the crypto transfer transaction")
-    public void verifyMirrorAPIResponses(int status) {
+    public void verifyMirrorAPICryptoTransferResponse(int status) {
         log.info("Verify transaction");
         String transactionId = networkTransactionResponse.getTransactionIdStringNoCheckSum();
-
         MirrorTransactionsResponse mirrorTransactionsResponse = mirrorClient.getTransactions(transactionId);
 
-        verifyCryptoTransfers(mirrorTransactionsResponse, status, transactionId);
-    }
-
-    private void verifyCryptoTransfers(MirrorTransactionsResponse mirrorTransactionsResponse, int status,
-                                       String transactionId) {
+        // verify valid set of transactions
         List<MirrorTransaction> transactions = mirrorTransactionsResponse.getTransactions();
         assertNotNull(transactions);
         assertThat(transactions).isNotEmpty();
 
+        // verify transaction details
         MirrorTransaction mirrorTransaction = transactions.get(0);
         if (status == HttpStatus.OK.value()) {
             assertThat(mirrorTransaction.getResult()).isEqualTo("SUCCESS");

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
@@ -20,19 +20,27 @@ package com.hedera.mirror.test.e2e.acceptance.steps;
  * ‍
  */
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.*;
 
+import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import io.cucumber.junit.platform.engine.Cucumber;
+import java.util.List;
 import lombok.Data;
 import lombok.extern.log4j.Log4j2;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 
 import com.hedera.hashgraph.sdk.AccountId;
 import com.hedera.hashgraph.sdk.Hbar;
-import com.hedera.hashgraph.sdk.TransactionReceipt;
 import com.hedera.mirror.test.e2e.acceptance.client.AccountClient;
+import com.hedera.mirror.test.e2e.acceptance.client.MirrorNodeClient;
+import com.hedera.mirror.test.e2e.acceptance.props.MirrorTransaction;
+import com.hedera.mirror.test.e2e.acceptance.props.MirrorTransfer;
+import com.hedera.mirror.test.e2e.acceptance.response.MirrorTransactionsResponse;
+import com.hedera.mirror.test.e2e.acceptance.response.NetworkTransactionResponse;
 
 @Log4j2
 @Cucumber
@@ -41,13 +49,20 @@ public class AccountFeature {
     private long balance;
     private AccountId accountId;
     private long startingBalance;
+    private NetworkTransactionResponse networkTransactionResponse;
 
     @Autowired
     private AccountClient accountClient;
 
+    @Autowired
+    private MirrorNodeClient mirrorClient;
+
     @When("I request balance info for this account")
     public void getAccountBalance() {
         balance = accountClient.getBalance();
+        log.info("{} account balance is {}",
+                accountClient.getSdkClient().getExpandedOperatorAccountId().getAccountId(),
+                balance);
     }
 
     @Then("the crypto balance should be greater than or equal to {long}")
@@ -61,31 +76,83 @@ public class AccountFeature {
         assertNotNull(accountId);
     }
 
+    @Given("I send {long} tℏ to {string}")
+    public void treasuryDisbursement(long amount, String accountName) {
+        accountId = accountClient
+                .getAccount(AccountClient.AccountNameEnum.valueOf(accountName)).getAccountId();
+
+        startingBalance = accountClient.getBalance(accountId);
+
+        networkTransactionResponse = accountClient
+                .sendCryptoTransfer(accountId, Hbar.fromTinybars(amount));
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
+    }
+
     @When("I send {long} tℏ to newly created account")
     public void sendTinyHbars(long amount) {
         startingBalance = accountClient.getBalance(accountId);
-        TransactionReceipt receipt = accountClient.sendCryptoTransfer(accountId, Hbar.fromTinybars(amount));
-        assertNotNull(receipt);
+        networkTransactionResponse = accountClient.sendCryptoTransfer(accountId, Hbar.fromTinybars(amount));
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
     }
 
     @When("I send {long} tℏ to account {int}")
     public void sendTinyHbars(long amount, int accountNum) {
         accountId = new AccountId(accountNum);
         startingBalance = accountClient.getBalance(accountId);
-        TransactionReceipt receipt = accountClient.sendCryptoTransfer(accountId, Hbar.fromTinybars(amount));
-        assertNotNull(receipt);
+        networkTransactionResponse = accountClient.sendCryptoTransfer(accountId, Hbar.fromTinybars(amount));
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
     }
 
     @When("I send {int} ℏ to account {int}")
     public void sendHbars(int amount, int accountNum) {
         accountId = new AccountId(accountNum);
         startingBalance = accountClient.getBalance(accountId);
-        TransactionReceipt receipt = accountClient.sendCryptoTransfer(accountId, Hbar.from(amount));
-        assertNotNull(receipt);
+        networkTransactionResponse = accountClient.sendCryptoTransfer(accountId, Hbar.from(amount));
+        assertNotNull(networkTransactionResponse.getTransactionId());
+        assertNotNull(networkTransactionResponse.getReceipt());
     }
 
     @Then("the new balance should reflect cryptotransfer of {long}")
     public void accountReceivedFunds(long amount) {
-        assertTrue(accountClient.getBalance(accountId) >= startingBalance + amount);
+        assertThat(accountClient.getBalance(accountId)).isGreaterThanOrEqualTo(startingBalance + amount);
+    }
+
+    @Then("the mirror node REST API should return status {int} for the crypto transfer transaction")
+    public void verifyMirrorAPIResponses(int status) {
+        log.info("Verify transaction");
+        String transactionId = networkTransactionResponse.getTransactionIdStringNoCheckSum();
+
+        MirrorTransactionsResponse mirrorTransactionsResponse = mirrorClient.getTransactions(transactionId);
+
+        verifyCryptoTransfers(mirrorTransactionsResponse, status, transactionId);
+    }
+
+    private void verifyCryptoTransfers(MirrorTransactionsResponse mirrorTransactionsResponse, int status,
+                                       String transactionId) {
+        List<MirrorTransaction> transactions = mirrorTransactionsResponse.getTransactions();
+        assertNotNull(transactions);
+        assertThat(transactions).isNotEmpty();
+
+        MirrorTransaction mirrorTransaction = transactions.get(0);
+        if (status == HttpStatus.OK.value()) {
+            assertThat(mirrorTransaction.getResult()).isEqualTo("SUCCESS");
+        }
+        assertThat(mirrorTransaction.getTransactionId()).isEqualTo(transactionId);
+        assertThat(mirrorTransaction.getValidStartTimestamp())
+                .isEqualTo(networkTransactionResponse.getValidStartString());
+        assertThat(mirrorTransaction.getName()).isEqualTo("CRYPTOTRANSFER");
+
+        assertThat(mirrorTransaction.getTransfers().size()).isGreaterThanOrEqualTo(3); // network, node and transfer
+
+        //verify transfer credit and debits balance out
+        long transferSum = 0;
+        for (MirrorTransfer cryptoTransfer : mirrorTransaction.getTransfers()) {
+            transferSum += Long.valueOf(cryptoTransfer.getAmount());
+        }
+
+        assertThat(transferSum).isEqualTo(0);
     }
 }

--- a/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
+++ b/hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java
@@ -153,6 +153,6 @@ public class AccountFeature {
             transferSum += Long.valueOf(cryptoTransfer.getAmount());
         }
 
-        assertThat(transferSum).isEqualTo(0);
+        assertThat(transferSum).isZero();
     }
 }

--- a/hedera-mirror-test/src/test/resources/features/account/account.feature
+++ b/hedera-mirror-test/src/test/resources/features/account/account.feature
@@ -6,8 +6,8 @@ Feature: Account Coverage Feature
         When I request balance info for this account
         Then the crypto balance should be greater than or equal to <threshold>
         Examples:
-            | threshold |
-            | 1000000   |
+            | threshold  |
+            | 1000000000 |
 
     @createcryptoaccount
     Scenario Outline: Create crypto account
@@ -16,3 +16,12 @@ Feature: Account Coverage Feature
         Examples:
             | amount    |
             | 100000000 |
+
+    @critical @release @acceptance @cryptotransfer
+    Scenario Outline: Validate simple CryptoTransfer
+        Given I send <amount> tℏ to <accountName>
+        Then the mirror node REST API should return status <httpStatusCode> for the crypto transfer transaction
+        And the new balance should reflect cryptotransfer of <amount>
+        Examples:
+            | amount | accountName | httpStatusCode |
+            | 1      | "ALICE"     | 200            |

--- a/hedera-mirror-test/src/test/resources/features/account/account.feature
+++ b/hedera-mirror-test/src/test/resources/features/account/account.feature
@@ -7,7 +7,7 @@ Feature: Account Coverage Feature
         Then the crypto balance should be greater than or equal to <threshold>
         Examples:
             | threshold  |
-            | 1000000000 |
+            | 4000000000 |
 
     @createcryptoaccount
     Scenario Outline: Create crypto account

